src/main/java/our/member/domain/TokenEntity.java

```diff
@@ -0,0 +1,62 @@
+package our.member.domain;
+
+import java.time.LocalDateTime;
+
+public class TokenEntity {
+    private Long tokenId;
+    private String token;
+    private Long userId;
+    private LocalDateTime createdAt;
+    private LocalDateTime expiresAt;
+
+    public TokenEntity() {
```

Comment:
기본 생성자 없이도 동작할 것 같아요! 그리고 만약 필요하다 해도 접근 제어자를 `public`으로 설정하면 의도하지 않은 곳에서 객체가 생성되고, 필요한 값을 받지 못할 위험이 있습니다.
```suggestion
    protected TokenEntity() {
```